manifests/pcap-api.yml

```diff
@@ -48,19 +51,23 @@ instance_groups:
             listen:
               port: 8080
               tls:
-                enabled: false
+                enabled: true
+                client_cas: ((pcap_api.client_cas))
+                certificate: ((pcap_api.certificate))
+                private_key: ((pcap_api.private_key))
             bosh:
               director_url: ((pcap_api.bosh_director_api))
               token_scope: bosh.admin
               agent_port: 9494
               tls:
-                common_name: bosh.service.cf.internal
+                enabled: true
+                common_name: ((pcap_api.bosh_director_name))
                 skip_verify: false
-                ca: ((/bootstrap-bosh/bosh/default_ca.ca))
+                ca: ((pcap_api.bosh_director_ca))
             agents_mtls:
               common_name: pcap-agent.service.cf.internal
               skip_verify: false
-              certificate: ((pcap_api_mtls.certificate))
+              certificate: ((pcap_api_mtls.certificate))((pcap_agent_mtls.ca))
               private_key: ((pcap_api_mtls.private_key))
               ca: ((pcap_agent_mtls.ca))
```

Comment:
We can use here the vars from vars-template.yml to be consistent.

Reply:
what do you mean? the `pcap_api_mtls` certificate has no definition in the vars-template.yml file.
The other entries are there only because they are not defined in the manifest and have to be imported from other deployments such as `cf`.

Reply:
you are right.